packages/a11y/src/index.js

```diff
@@ -0,0 +1,46 @@
+import addContainer from './addContainer';
+import clear from './clear';
+import domReady from '../../domReady/src/';
```

Comment:
Not even sure if this will go well when publishing to NPM. Maybe that'd require me to first do separate PR for the domReady package.